docs/casa/plugins/accts-linking/accts-linking-agama.md

```diff
@@ -0,0 +1,87 @@
+# Accounts linking project configuration
+
+## Overview
+
+The accounts linking Agama project must be configured in order to integrate third-party identity providers. The configuration of this project is supplied in a JSON file whose structure is like:
+
+```
+{
+"io.jans.casa.acctlinking.Launcher": {
+    
+    "providerID_1": { ... },
+    "providerID_2": { ... },
+    ...
+}
+}
+```
+
+Each property part of the JSON object `io.jans.casa.acctlinking.Launcher` holds the configuration of a different identity provider. Here's a how a typical configuration of a provider looks like:
+
+```
+{
+    "displayName": "Goooogle",
+    "flowQname": "io.jans.inbound.GenericProvider",
+    "mappingClassField": "io.jans.casa.acctlinking.Mappings.GOOGLE",    
+    "oauthParams": {
+        "authzEndpoint": "https://accounts.google.com/o/oauth2/v2/auth",
+        "tokenEndpoint": "https://oauth2.googleapis.com/token",
+        "userInfoEndpoint": "https://www.googleapis.com/oauth2/v3/userinfo",
+        "clientId": "202403151302",
+        "clientSecret": "m|a1l2d3i4t5a6S7h8a9k0i'r¿a",
+        "scopes": ["email", "profile"]
+    }
+}
+   
+```         
+
+In this case, we are populating the configuration of an OAuth-based provider called "Goooogle". 
+
+The tables shown in [this](https://github.com/JanssenProject/jans/blob/main/docs/agama-catalog/jans/inboundID/README.md#supply-configurations) page list all possible properties to configure a provider. Particularly, two properties deserve the most detail:
```

Comment:
This link should be a https://docs.jans.io . @ossdhaval this has to be added to the mkdocs and then referenced here.